platform/ios/src/NSOrthography+MGLAdditions.h

```diff
@@ -0,0 +1,7 @@
+#import <Foundation/Foundation.h>
+
+@interface NSOrthography (NSOrthography_MGLAdditions)
+
++ (NSString *)mgl_dominantScriptForMapboxStreetsLanguage:(NSString *)language;
```

Comment:
Document that this method takes a locale identifier and returns an ISO 15924 script code.